### PR TITLE
Rewrite intro, reorganize text in discovery section

### LINKED ISSUE #52: Circumstances under which profiles are used or shared
WebID identifies one within the space of social, conceptual, and contextual relations.

Where possible, the Solid WebID Profile specification should aim to provide recommendations; clarify and open the space for recommendations to be developed; waypoint to existing documentation; or include considerations for circumstances under which profiles are used or shared, e.g.:

* [ ] Communication and use of multiple profiles for different purposes.
* [ ] Connectivity of multiple profiles and cross-context recognition.
* [ ] Cascade of profile information.
* [ ] Persistence and evolution of profiles.
* [ ] Anonymous and pseudonymous profiles.
* [ ] Self-controlled and third-party-controlled identities.
* [ ] Information for the purpose of verification, authentication, authorization purposes.
* [ ] Compilation of preferences, choices, activities and interactions of an agent.
* [ ] Social graph for different kinds of relationships between agents.
* [ ] Factors specific to physical, physiological, genetic, mental, economic, cultural, social, or behavioural identity.

### PR DESCRIPTION
**Pull request by VirginiaBalseiro**
This PR is a rewrite of the introduction. 

Content about discovery has been moved to the discovery section (also to be revised soon).

Closes #52 
Partially addresses #55 

[Preview](https://htmlpreview.github.io/?https://raw.githubusercontent.com/solid/webid-profile/5e498bcaca0a9bc092a06f81a6a8de2c441257ba/index.html) | [Diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fsolid.github.io%2Fwebid-profile%2F&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fwebid-profile%2F5e498bcaca0a9bc092a06f81a6a8de2c441257ba%2Findex.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1209,6 +1115,29 @@ <h2 property="schema:name">Discovering a complete Solid profile</h2>
...
+                Profiles can contain, and apps are free to follow, any kind of
+                links to related documents. In order to promote interoperability
+                and limit the burden on apps, this specification recommends a
+                limited number of related documents which a profile ought to
+                contain so that apps can discover them.
+              </p>
+              <p>
+                The discovery process starts with the WebID, a URI that points
+                to exactly one document, referred to here as a WebID Profile
+                Document. This document can be expected to contain pointers to a
```
Comment (csarven) [inline index.html:1128]:
```suggestion
                The discovery process starts with the WebID, a URI that points
                to exactly one document, referred to here as a WebID Profile
                Document [<cite><a class="bibref" href="#bib-webid">WEBID</a></cite>]. This document can be expected to contain pointers to a
```

And update similarly if need to be clear.

File: index.html [hunk 2]
```diff
@@ -1209,6 +1115,29 @@ <h2 property="schema:name">Discovering a complete Solid profile</h2>
...
+                Preferences File Document containing settings &amp; resources
```
Comment (csarven) [inline index.html:1129]:
```suggestion
                Preferences File Document containing settings and resources
```

File: index.html [hunk 3]
```diff
@@ -1209,6 +1115,29 @@ <h2 property="schema:name">Discovering a complete Solid profile</h2>
...
+                meant only for the WebID owner, Type Index Documents containing
+                links to specific types of resources, and might also contain
+                Extended Profile Documents containing additional information
```
Comment (csarven) [inline index.html:1132]: Same as earlier.. emphasise or link.

**Review by csarven — APPROVED** [review]
Let's make sure now to lose comments / use cases like in the mix of closing issues when merging, e.g., https://github.com/solid/webid-profile/issues/55#issuecomment-1294210891 , and that the specification is able to provide a solution for it.

File: index.html [hunk 4]
```diff
@@ -848,104 +847,31 @@ <h2 id="table-of-contents">Table of Contents</h2>
...
+                <li>Connectivity of multiple profiles and cross-context recognition.</li>
+                <li>Cascade of profile information.</li>
+                <li>Persistence and evolution of profiles.</li>
+                <li>Anonymous and pseudonymous profiles.</li>
+                <li>Self-controlled and third-party-controlled identities.</li>
+                <li>Information for the purpose of verification, authentication, authorization purposes.</li>
+                <li>Compilation of preferences, choices, activities and interactions of an agent.</li>
+                <li>Social graph for different kinds of relationships between agents.</li>
+                <li>Factors specific to physical, physiological, genetic, mental, economic, cultural, social, or behavioural identity.
+                </li>
```
Comment (TallTed) [inline index.html:870]:
```suggestion
                <li>Factors specific to physical, physiological, genetic, mental, economic, cultural, social, or behavioural identity.</li>
```

**Pull request merged**